main.js

```diff
@@ -1,53 +1,176 @@
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
+      medium: 'Medium: 10x10 inches Unframed Print (approx 25.4cm sq.)',
+      large: 'Large: 12x12 inches Unframed Print (approx 30.4cm sq.)'
+    },
+    price: 'Small: $16.99 | Medium: $19.99 | Large: $22.99',
+    id: 7
+  },
+  {
+    image: 'images/chiaroscuro.jpg',
+    title: 'Chiaroscuro',
+    description: 'This original signed photograph is professionally printed at a lab on archival photography paper.',
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
+      medium: 'Medium: 10x10 inches Unframed Print (approx 25.4cm sq.)',
+      large: 'Large: 12x12 inches Unframed Print (approx 30.4cm sq.)'
+    },
+    price: 'Small: $16.99 | Medium: $19.99 | Large: $22.99',
+    id: 8
+  },
+  {
+    image: 'images/valiant.jpg',
+    title: 'Valiant Vocalist',
+    description: 'This original signed photograph is professionally printed at a lab on archival photography paper.',
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
+      medium: 'Medium: 10x10 inches Unframed Print (approx 25.4cm sq.)',
+      large: 'Large: 12x12 inches Unframed Print (approx 30.4cm sq.)'
+    },
+    price: 'Small: $16.99 | Medium: $19.99 | Large: $22.99',
+    id: 9
+  },
+  {
+    image: 'images/energetic.jpg',
+    title: 'Energetic Enigma',
+    description: 'This original signed photograph is professionally printed at a lab on archival photography paper.',
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
+      medium: 'Medium: 10x10 inches Unframed Print (approx 25.4cm sq.)',
+      large: 'Large: 12x12 inches Unframed Print (approx 30.4cm sq.)'
+    },
+    price: 'Small: $16.99 | Medium: $19.99 | Large: $22.99',
+    id: 10
+  },
+  {
+    image: 'images/curiosity.jpg',
+    title: 'Capricious Curiosity',
+    description: 'This original signed photograph is professionally printed at a lab on archival photography paper.',
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
+      medium: 'Medium: 10x10 inches Unframed Print (approx 25.4cm sq.)',
+      large: 'Large: 12x12 inches Unframed Print (approx 30.4cm sq.)'
+    },
+    price: 'Small: $16.99 | Medium: $19.99 | Large: $22.99',
+    id: 11
   }
 ]
 
+var $siteDescription = document.querySelector('.site-description')
 var $artworkSection = document.querySelector('.artwork')
+var $product = document.getElementsByClassName('product')
+var $image = document.getElementsByClassName('artwork-image')
+var $productDescription = document.getElementsByClassName('product-description')
+var $back = document.querySelector('.back')
 
 document.addEventListener('DOMContentLoaded', function () {
   renderPhotos(products)
+  addEventsToArtwork($image)
+  addEventsToArtwork($productDescription)
 })
 
-function createEachProduct(products) {
+function createEachProduct(products, productId) {
```

Comment:
The parameter should be `product` because it is singular if it was more than one it would be products.